### PR TITLE
Update to SRTM download, documentation update

### PR DESCRIPTION
**Pull request by dafenner**
- updated automatic SRTM download in cqcp_add_dem_height/cqcp_download_srtm with the `geodata` package instead of `raster` package (#27).
- included published time constants in documentation of `cqcp_o4` (#25)

### CODE REVIEW

**Comment by dafenner** [comment]
@theendlessriver13 
Pull request for updated package v1.0.1

File: R/cqcp_helper.R [hunk 1]
```diff
@@ -83,7 +83,7 @@ cqcp_check_input <- function(data, print = TRUE, file = NULL, as_list = FALSE){
   if(!has_z) {
     miss_1b <- c(miss_1b, "z")
     mess_1b <- cqcp_colourise("     ! Missing: z\n", "yellow")
-    mess_1b <- cqcp_colourise("     --> Filters cqcp_m2 and cqcp_m5 will not work with 'heightCorrection = T'. You can run 'cqcp_add_dem_height' to add DEM information.\n", "yellow")
+    mess_1b <- c(mess_1b, cqcp_colourise("     --> Filters cqcp_m2 and cqcp_m5 will not work with 'heightCorrection = T'. You can run 'cqcp_add_dem_height' to add DEM information.\n", "yellow"))
```
Comment (theendlessriver13) [inline R/cqcp_helper.R:86]: is `mess_1b` intentional? or should it be `miss_1b`?

**Review by theendlessriver13 — COMMENTED** [review]
looks good! just one question regarding a variable name (see below)

File: R/cqcp_helper.R (hunk 1 above)
Comment (dafenner) [reply R/cqcp_helper.R:86]: Yes, intentional.
`mess_1b` for message, `miss_1b` for what is missing.

**Review by theendlessriver13 — APPROVED** [review]

**Pull request merged**